source/rst/lss_examples.rst

```diff
@@ -348,7 +349,7 @@ a one standard-deviation investment shock in our model
     plt.title('Impulse response to positive investment shock with $b = 0.5$', fontsize=14);
```

Comment:
can we delete all the fontsize arguments? I've tried to avoid them as I think they're unncessary